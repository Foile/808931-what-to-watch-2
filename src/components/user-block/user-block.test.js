import React from 'react';
import renderer from "react-test-renderer";
import UserBlock from './user-block';
import {BrowserRouter} from "react-router-dom";

it(`UserPage correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<BrowserRouter>
    <UserBlock
      onSubmit = {jest.fn()}
      isAuthorizationRequired = {true}
    /></BrowserRouter>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
