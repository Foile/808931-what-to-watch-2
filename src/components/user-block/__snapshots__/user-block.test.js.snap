// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPage correctly renders after relaunch 1`] = `
<div
  className="user-block"
>
  <a
    href="/login"
    onClick={[Function]}
  >
    Sign in
  </a>
</div>
`;
